addons/l10n_it_edi/tests/test_edi_export.py

```diff
@@ -43,6 +43,17 @@ def setUpClass(cls):
             'is_company': True,
         })
 
+        # ==== Cash rounding ====
```

Comment:
No need for the comment, there is none in the big setUpClass function for all the rest we create in advance ^^
The following line is literally `cls.cash_rounding_a = <stuff>`